app/index.ts

```diff
@@ -102,18 +102,29 @@ class GiuseppeGenerator extends yeomanGenerator.Base {
         this.log(`${chalk.blue('Install')} - installing dependencies ${chalk.dim('(npm and typings)')}.`);
 
         let deps = ['giuseppe', 'express', 'body-parser'],
-            devDeps = ['del-cli', 'typescript', 'typings', 'tslint'],
-            globalTypings = ['dt~express', 'dt~express-serve-static-core', 'dt~http-status', 'dt~mime', 'dt~node', 'dt~serve-static', 'dt~body-parser'],
+            devDeps = [
+                'del-cli',
+                'typescript',
+                'typings',
+                'tslint',
+                '@types/express',
+                '@types/express-serve-static-core',
+                '@types/http-status',
+                '@types/mime',
+                '@types/node',
+                '@types/serve-static',
+                '@types/body-parser'
+                ],
             devTypings = [],
```

Comment:
Those should be unused variables now right? (`devTypings` and `devGlobalTypings`)